src/content/en/faq/import-failed.md

```
@@ -0,0 +1,44 @@
+slug: import-failed
+navId: import-failed
+title: "(IMPORT FAILED)"
+summary: "Custom node loading failure"
+permalink: "/{{ lang }}/{{ section }}/{{ slug }}/"
+hero:
+  image: ""
+---
+
+## Symptom
+
+![](https://gyazo.com/3054a82b909490117748ae061e29c35e){gyazo=image}
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)